src/tests/multihost/alltests/test_misc.py

```diff
@@ -130,10 +130,15 @@ def test_0003_sssd_crashes_after_update(self, multihost,
         domain_name = tools.get_domain_section_name()
```

Comment:
We have this domain with parameter.
On line 135, why are you creating again the another domain?

Reply:
tl;dr this testcase requires at least two domains.

At the beginning, class-fixture "setup_sssd" configures ldap-domain. Here [L130](https://github.com/SSSD/sssd/pull/7196/files#diff-b73320f0c0cd4f7e352ac3bf59eda15199fe60b70610b5bfaf29266c2a1b74d3L130) contains this  ldap-server value. The 'enable_files_domain = true' was adding/creating an implicit domain for files in sssd.conf

> 
      From man sssd.conf 
      enable_files_domain (boolean)
           When this option is enabled, SSSD prepends an implicit domain with “id_provider=files” before any explicitly configured domains.

           Default: false 

Now that 'files-provider' is being translated with proxy-provider, it needs to be set up explicitly. Thus a separate section(local) is being added.

Reply:
okay, we have two domain here,

[domain/<domain1>]
...
...

[domain/<domain2>]
...
...

for <domain2> we have **id_provider = proxy** but what will the id_provider or related parameter for <domain1>?